src/Akka.Hosting/ActorRegistry.cs

```diff
@@ -160,6 +183,13 @@ public interface IReadOnlyActorRegistry : IEnumerable<KeyValuePair<Type, IActorR
     /// </remarks>
     public interface IActorRegistry: IReadOnlyActorRegistry
     {
+        /// <summary>
+        /// Registers an actor into the registry. Throws an exception upon failure.
+        /// </summary>
+        /// <param name="actor">The bound <see cref="IActorRef"/>, if any. Is set to <see cref="ActorRefs.Nobody"/> if key is not found.</param>
```

Comment:
So we can register a null value to a key?

Reply:
You can - nothing preventing that for now.

Reply:
Although this XML-DOC is wrong.

Reply:
Might want to do a null check and throw a `NullArgumentException`, maybe?

Reply:
Good idea